artemis-jms-client/pom.xml

```diff
@@ -50,6 +50,11 @@
          <artifactId>artemis-core-client</artifactId>
          <version>${project.version}</version>
       </dependency>
+      <dependency>
+         <groupId>org.apache.activemq</groupId>
+         <artifactId>artemis-commons</artifactId>
```

Comment:
Why is this needed it comes as a transitive anyhow

Reply:
Yes, it comes in as transitive, but `org.apache.activemq.artemis.jms.client.ActiveMQDestination` uses `org.apache.activemq.artemis.api.core.QueueConfiguration` (from `artemins-commons`) explicitly so IMO the dependency should be explicit as well instead of leaving it as transitive.

Reply:
Fair enough +1